### DISCUSSION TITLE
DateTime picker

### DISCUSSION THREAD

**Discussion by iDigAcademy** [laravel/nova-issues#4892] [Q&A] [answered]
Nova v4 using DateTime field.

Just a quick question: Is there suppose to be a time in the date picker that pops up? Because all I see is the calendar in the popup. I do see where a user can enter the time by hand in the field.

**Comment by crynobone** [comment, marked as answer]
> Because all I see is the calendar in the popup. I do see where a user can enter the time by hand in the field.

Each browser implements the feature differently.
